db/models/articles.models.js

```diff
@@ -36,7 +36,14 @@ exports.fetchAllArticles = (sort_by = "created_at", order = "DESC", topic) => {
 
 exports.fetchArticleByID = (article_id) => {
   return db
-    .query(`SELECT * FROM articles WHERE article_id = $1;`, [article_id])
+    .query(
+      `SELECT articles.author, articles.title, articles.body, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, CAST (COUNT (comment_id) AS INT) AS comment_count
```

Comment:
I believe you can still use the star and then add a comma after that and add the comment count?

Reply:
`SELECT articles.*, CAST(COUNT(comment_id) AS INT) AS comment_count`